cmd/minikube/cmd/start.go

```diff
@@ -102,6 +103,15 @@ func runStart(cmd *cobra.Command, args []string) {
 	kubeHost = strings.Replace(kubeHost, ":2376", ":"+strconv.Itoa(constants.APIServerPort), -1)
 	fmt.Printf("Kubernetes is available at %s.\n", kubeHost)
 
+	//Checking for kubectl
+	fmt.Printf("Checking for Kubectl ...\n")
+	path, err := exec.LookPath("kubectl")
+	if err != nil {
+		log.Fatal("Kubectl is not installed.")
```

Comment:
Thanks @jimmidyson for the suggestion.I will add some nice suggestion for kubectl.